### DISCUSSION TITLE
Text extraction position is incorrect

### DISCUSSION THREAD

**Discussion by osiast** [mindee/doctr#1173] [Q&A] [answered]
Hi,
I'm using the latest version of Doctr (v0.6.0) to extract text from PDF documents. However, I noticed that the extracted text is often positioned slightly lower than the original text in the document.

![image](https://user-images.githubusercontent.com/2292984/231759239-6926203b-f824-408a-b808-2d7d4296baa2.png)
![image](https://user-images.githubusercontent.com/2292984/231759444-bed08231-0096-453c-a111-03a160cbf839.png)

I have reviewed the code that calls the export_pdfa function and the functions _get_baseline and _pt_from_pixel, but I did not find any obvious errors.

Could you please provide me suggestion for resolving this problem?

Thank you.

**Comment by felixdittrich92** [comment, marked as answer]
Hi @osiast :wave:,
you are right the tutorials needs some updates.
As an alternative you can combine docTR with OCRmyPDF @SiddhantBahuguna wrote a blog post a while ago: https://mindee.com/blog/create-ocrized-pdfs-in-2-steps/
